pkgs/development/python-modules/netcdf4/skipDapTest.patch

```diff
@@ -0,0 +1,60 @@
+Only in netCDF4-1.5.3.new: build
+Only in netCDF4-1.5.3.new: dist
+Only in netCDF4-1.5.3.new/include: constants.pyx
+Only in netCDF4-1.5.3.new: .mypy_cache
+Only in netCDF4-1.5.3.new/netCDF4: _netCDF4.c
+diff -cr netCDF4-1.5.3/netCDF4.egg-info/SOURCES.txt netCDF4-1.5.3.new/netCDF4.egg-info/SOURCES.txt
+*** netCDF4-1.5.3/netCDF4.egg-info/SOURCES.txt	2019-10-21 18:38:19.000000000 +0200
+--- netCDF4-1.5.3.new/netCDF4.egg-info/SOURCES.txt	2020-05-28 00:11:43.559717088 +0200
+***************
+*** 3,8 ****
+--- 3,9 ----
+  Changelog
+  MANIFEST.in
+  README.md
++ nix_run_setup
+  setup.cfg
+  setup.py
+  docs/index.html
+***************
+*** 28,33 ****
+--- 29,35 ----
+  man/nc4tonc3.1
+  man/ncinfo.1
+  netCDF4/__init__.py
++ netCDF4/_netCDF4.c
+  netCDF4/_netCDF4.pyx
+  netCDF4/utils.py
+  netCDF4.egg-info/PKG-INFO
+Only in netCDF4-1.5.3.new: nix_run_setup
+Only in netCDF4-1.5.3.new/test: __pycache__
+diff -cr netCDF4-1.5.3/test/tst_dap.py netCDF4-1.5.3.new/test/tst_dap.py
+*** netCDF4-1.5.3/test/tst_dap.py	2019-10-21 18:38:12.000000000 +0200
```

Comment:
This patch is messy. Please remove it. You probably should be able to achieve the same effect with "rm test/tst_dap.py". Would be better to switch to `pytestCheckHook` and put those in `disabledTests`.

Reply:
I have used the rm version, since I had trouble using pytest (which I tried first).
I observed some very strange behavior that I could not understand at all:
When I used nix-shell --pure the pytest was working but with nix-build they never got further than starting to collect. I could not see the actual error message. 
This is the reason I switched to unittest.
This is very esoteric and for my further endeavors in the nix world I would be very happy to get a clue what exactly is different in the nix-shell env. I have hunted for shell variables but could not explain the different outcome.

Reply:
Sorry to bother you.
I am not sure how to proceed. I think I have incorporated everything that had been demanded.
I would like to finish this.
Is there something still blocking?
Thanks in advance 
Markus